Backend/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "lbtc-be",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "bcryptjs": "2.4.3",
+    "cors": "2.8.5",
```

Comment:
Update all dependencies, there are several security advisories showing up if you execute `npm audit`